src/obr/core/queries.py

```diff
@@ -69,6 +69,9 @@ def execute(self, key, value):
 
         # case: specific value, existent key. This is effectively a filter.
         try:
+            # NOTE this sadly doesnt work for e.g. NoSubdomains, when self.value==0.25 and value is 1
```

Comment:
It CAN, because there is no check during the initialization whether or not the environment variables are valid. But I agree that this is not an issue at this point but rather during the yaml parsing.

Reply:
Ok fair enough, but maybe replace NoSubdomains by just value, because requesting numberOfSubDomains to be < 1 does not make any sense.